### PR TITLE
Provide a better fallback message if Rekognition is not available

### LINKED ISSUE #112: Better messaging when AWS Rekognition is non operational
On a local Altis development environment, AWS Rekognition appears to be non operational. This results in either an infinite spinner for image labels or a plain blank meta box. It would be better if there was an informational message shown in place of the labels when Rekognition can't do its thing.

### PR DESCRIPTION
**Pull request by roborourke**
Previously the spinner would show and then an error message on reload.

Fixes https://github.com/humanmade/altis-media/issues/112

Shows the following instead:

Screenshot 2021-06-21 at 15 46 32

### CODE REVIEW

**Review by jazzsequence — COMMENTED** [review]
Aside from the linting issues, this looks good and an improvement on what's there now. Is it possible to hide the "Detected image labels" label if Rekognition is unavailable?

Another question -- is Rekognition specifically named anywhere in our documentation? If not, we might want to just say "Image recognition is unavailable".

**Comment by roborourke** [comment]
Yeah we could just hide it altogether, I wondered about that vs still showing that something would come up if the feature was available.

I'll update the text to your suggestion too 👍

File: inc/admin/namespace.php [hunk 1]
```diff
@@ -42,17 +42,26 @@ function attachment_fields( $fields, WP_Post $post ) {
...
 function get_keywords_html( $post_id, $limit = 10 ) : string {
+	$error_markup = '<style>
+		.compat-field-hm-aws-rekognition-labels p { margin: 6px 0; }
+	</style>
+	<p>%s</p>';
+
+	if ( ! AWS_Rekognition\is_available() ) {
+		return sprintf(
+			$error_markup,
+			esc_html__( 'Rekognition is currently unavailable.', 'hm-aws-rekognition' )
```
Comment (roborourke) [inline inc/admin/namespace.php]:
```suggestion
			esc_html__( 'Image recognition is currently unavailable.', 'hm-aws-rekognition' )
```

**Comment by jazzsequence** [comment]
I think that if we're explicitly saying "Image recognition is not available" the implication is that there will be times when it _is_ available. "Detected image labels: Image recognition is not available" gives the impression that there's an error or a problem when generally it's just that you're not connected to AWS services (which, I suppose, on a prod environment _would_ be a problem, but otherwise is expected).

However, I think this is a good start and we can come back to that.

**Review by jazzsequence — APPROVED** [review]

**Pull request merged**